apps/admin/backend/src/server.ts

```diff
@@ -38,11 +38,22 @@ function constructDippedSmartCardAuthMachineState(
   workspace: Workspace
 ): DippedSmartCardAuthMachineState {
   // TODO: Once we actually support multiple elections, configure the auth instance with the
-  // currently selected election rather than the first. In fact, do so as soon as the currently
-  // selected election is persisted on the backend instead of the frontend since, even today, in
-  // dev, we can end up with multiple election definitions under the hood via incognito windows
+  // currently selected election rather than the most recently created. In fact, do so as soon as
+  // the currently selected election is persisted on the backend instead of the frontend since,
+  // even today, in dev, we can end up with multiple election definitions under the hood via
+  // incognito windows
```

Comment:
This was an interesting one. Since Cypress tests do trigger this "multiple election definitions under the hood" state, but we no longer set the auth lib election definition when a user loads an election definition (every integration test starts with the load of an election definition), I had to update the selection logic here to select the most recently created election definition rather than the first

Again, all inconsequential for prod

Reply:
Is there any way to make the Cypress tests unconfigure when they are done so that we don't have this problem?

Reply:
Went ahead and added this to my [miscellaneous cleanup issue](https://github.com/votingworks/vxsuite/issues/2965). I think a generalized solution for resetting the backend between Cypress tests would be great